src/app/educations/page.tsx

```diff
@@ -0,0 +1,243 @@
+            ([key, value]) =>
+              education[key as keyof Education]
+                ?.toString()
+                .toLowerCase()
+                .includes(value.toLowerCase())
+          );
+        });
+      }
+
+      setTotalPageNumber(Math.ceil(filteredData.length / 10));
+      setData(filteredData.slice(page * 10, (page + 1) * 10));
+    } catch (err) {
+      setError("Error during data loading");
+    } finally {
+      setIsLoading(false);
+    }
+  }, [page, debouncedSearchTerm, educations]);
+
+  useEffect(() => {
+    saveEducationsToStorage(educations);
+  }, [educations]);
+
+  const onSearch = (key: string, value: string) => {
+    setFilter((prevFilter) => ({ ...prevFilter, [key]: value }));
+    setPage(0);
+  };
+
+  const addEducation = () => {
+    if (newEducationTitle.trim() === "" || newEducationDescription.trim() === "") {
+      alert("Molim, unesite naziv i opis novog događaja.");
+      return;
+    }
+  
+    const newEducation: Education = {
+      id: educations.length + 1,
+      title: newEducationTitle || "",
+      date: new Date().toISOString().split("T")[0] || "",
+      description: newEducationDescription || "",
+      active: false,
+    };
+  
+    setEducations((prevEducations) => [...prevEducations, newEducation]);
+    setNewEducationTitle(""); 
+    setNewEducationDescription(""); 
+  };
+  
+
+  const deleteEducation = (id: number) => {
+    setEducations(educations.filter((education) => education.id !== id));
+  };
+
+  const toggleEducationStatus = (id: number) => {
+    setEducations(
+      educations.map((education) =>
+        education.id === id
+          ? { ...education, active: !education.active }
+          : education
+      )
+    );
+  };
+
+  const openEducationModal = (education: Education) => {
+    setSelectedEducation(education);
+  };
+
+  const closeEducationModal = () => {
+    setSelectedEducation(null);
+  };
+
+  return (
+    <MainLayout headerChildren={<div>Educations</div>}>
+      <main className="flex flex-1 flex-col gap-4 p-4 md:gap-8 md:p-6">
+        <div className="flex gap-5">
+          <SearchInput title={"Naziv"} onSearch={onSearch} searchKey={"title"} />
+          <SearchInput title={"Datum"} onSearch={onSearch} searchKey={"date"} />
+        </div>
+        <div className="flex flex-col gap-4">
+          <input
+            type="text"
+            placeholder="Naziv novog događaja"
+            value={newEducationTitle}
+            onChange={(e) => setNewEducationTitle(e.target.value)}
+            className="p-2 border rounded-md"
+          />
+          <textarea
+            placeholder="Opis novog događaja"
+            value={newEducationDescription}
+            onChange={(e) => setNewEducationDescription(e.target.value)}
+            className="p-2 border rounded-md"
+          />
+          <button onClick={addEducation} className="btn-primary">
+            Dodati događaj
+          </button>
+        </div>
```

Comment:
This div is not part of this ticket, and will be used differently so it can be removed